docs/RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+# Go-IPFS Changelog Template
+
+## X.Y.Z YYYY-MM-DD
+
+We're happy to announce go-ipfs X.Y.Z, bla bla...
+
+< release notes >
+
+### Contributors
+
+< list generated by bin/mkreleaselog >
+
+Would you like to contribute to the IPFS project and don't know how? Well, there are a few places you can get started:
+
+- Check the issues with the `help wanted` label in the [go-ipfs repo](https://github.com/ipfs/go-ipfs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
+- Join an IPFS All Hands, introduce yourself and let us know where you would like to contribute - https://github.com/ipfs/team-mgmt/#weekly-ipfs-all-hands
+- Hack with IPFS and show us what you made! The All Hands call is also the perfect venue for demos, join in and show us what you built
+- Join the discussion at http://discuss.ipfs.io/ and help users finding their answers.
+- Join the [Go Core Dev Team Weekly Sync](https://github.com/ipfs/team-mgmt/issues/650) and be part of the Sprint action!
+
+### ⁉️ Do you have questions?
+
+The best place to ask your questions about IPFS, how it works and what you can do with it is at [discuss.ipfs.io](http://discuss.ipfs.io). We are also available at the `#ipfs` channel on Freenode, which is also [accessible through our Matrix bridge](https://riot.im/app/#/room/#freenode_#ipfs:matrix.org).
+
+### Changelog
+
+< changelog generated by bin/mkreleaselog >
```

Comment:
This should come before the Contributors section.

Reply:
I'm worried about burying the contributors. This is what an itemized changelog looks like for go-ipfs: https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md#changelog